website/react-magma-docs/src/pages/api/indeterminate.mdx

```diff
@@ -0,0 +1,89 @@
+---
+pageTitle: Indeterminate Checkbox API
+title: Indeterminate Checkbox
+props:
+  - IndeterminateCheckboxProps
+---
+
+<DocsHeading to="/design/selection-controls" type="api">
+  Indeterminate
+</DocsHeading>
+
+The `IndeterminateCheckbox` component is similar to the `Checkbox` component, except that instead of accepting a boolean `checked` value,
```

Comment:
Let's make the words "Checkbox component" a link to the checkbox page of the docs.